.github/docker/Dockerfile.centreon-collect-debian-bookworm

```diff
@@ -0,0 +1,68 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/debian:bookworm
+
+RUN <<EOF
+
+apt-get update
+
+echo 'http://deb.debian.org/debian' | apt-get install -y pbuilder
```

Comment:
I guess pbuilder is not required since we use nfpm

Reply:
https://github.com/centreon/centreon-collect/actions/runs/7568830922/job/20610838426 i wouldn't be so sure about this

Reply:
it fails because you install `dh-make` , `lintan`, `quilt`, `git-buildpackage` and `devscripts` and `debmake` too
all those packages should be useless

Reply:
oh makes sense